i18n/pt-br/code.json

```diff
@@ -0,0 +1,150 @@
+    "message": "Compatibilidade",
+    "description": "Reasons list item title"
+  },
+  "reasons.item-list.6.content": {
+    "message": "Nós suportamos SCORM e xAPI, para facilitar a migração, além do nosso formato próprio, chamado: Guaraná.",
+    "description": "Reasons list item content"
+  },
+  "features.item-list.1.toptitle": {
+    "message": "Espaço do Estudante",
+    "description": "Feature list item toptitle"
+  },
+  "features.item-list.1.title": {
+    "message": "Um espaço de aprendizado único",
+    "description": "Feature list item title"
+  },
+  "features.item-list.1.description": {
+    "message": "O estudante pode customizar a forma com que ele acessa seus cursos, baseado em seus interesses. Enquanto você aprende, a plataforma também aprenderá com você e se adaptará para para ajudar você a melhorar seu desempenho. O Espaço do Estudante é perfeito para melhorar a forma que você aprende enquanto monitora seu progresso e discute com instrutores e outros estudantes.",
+    "description": "Feature list item description"
+  },
+  "features.item-list.2.toptitle": {
+    "message": "Espaço do Instrutor",
+    "description": "Feature list item toptitle"
+  },
+  "features.item-list.2.title": {
+    "message": "Um espaço para ensinar",
+    "description": "Feature list item title"
+  },
+  "features.item-list.2.description": {
+    "message": "O controle do conteúdo em suas mãos. Obtenha estatísticas sobre o desempenho e progresso de seus alunos, turmas e cursos. Por meio de análises de máquina, encontre formas de melhorar a forma de ministrar suas suas e suprir a necessidade de individual de cada aluno.",
+    "description": "Feature list item description"
+  },
+  "features.item-list.3.toptitle": {
+    "message": "Para organizações",
+    "description": "Feature list item toptitle"
+  },
+  "features.item-list.3.title": {
+    "message": "Setor Público ou Privado",
+    "description": "Feature list item title"
+  },
+  "features.item-list.3.description": {
+    "message": "Distribua conhecimento como parte do seu negócio. Encontre novas formas de ensinar, aprender, compartilhar e colaborar. Gerencie vários espaços de aprendizado, e ajuda seus colaboradores ou estudantes em sua jornada pelo conhecimento.",
+    "description": "Feature list item description"
+  },
+  "homepage.section.reasons.title": {
+    "message": "Por que Sumé LMS?",
+    "description": "Homepage Reasons section title"
+  },
+  "homepage.section.install.title": {
+    "message": "Instale em todo lugar",
+    "description": "Homepage Installs section title"
+  },
+  "homepage.section.install.subtitle": {
+    "message": "Alguns guidas para ajudar você a instalar e configurar a plataforma.",
+    "description": "Homepage Installs section subtitle"
+  },
+  "homepage.section.install.button": {
+    "message": "Começar agora",
+    "description": "Homepage Install section button"
+  },
+  "homepage.section.community.title": {
+    "message": "Faça a Evolução",
```

Comment:
Faça a Evolução?

Reply:
Então, em inglês tá "Do the evolution", na real é uma referência a PJ :)

Reply:
@arianerocha alguma sugestão para frase melhor em ptbr e inglês?

Reply:
Quando bati o olho eu li `Faça a revolução`  :fist_raised: 
A princípio não tenho sugestão, mas se conseguir pensar em algo eu jogo pra discussão.